doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -713,8 +713,65 @@ so notification receiver can know the format used without needing processing not
 
 ### Notification templates
```

Comment:
Fixed in 9585da0

I'm using a new name for the section: "Custom notifications"